modules/rup/internacion/cama-estados.controller.ts

```diff
@@ -10,13 +10,17 @@ export async function snapshotEstados({ fecha, organizacion, ambito, capa }, fil
     const firstMatch = {};
     const secondMatch = {};
     const thirdMatch = {};
+    const ambitoMatch = {};
     if (filtros.cama) {
         firstMatch['idCama'] = mongoose.Types.ObjectId(filtros.cama);
     }
 
     if (filtros.paciente) {
         secondMatch['paciente.id'] = mongoose.Types.ObjectId(filtros.paciente);
     }
+    if (ambito) {
+        ambitoMatch['ambito'] = ambito;
```

Comment:
No sería mas conveniente agregar el filtro a `firstMatch` en lugar de agregar un nuevo match aparte?
```suggestion
        firstMatch['ambito'] = ambito;
```

Reply:
@waltergarrido revisar comentario

Reply:
@martinebucarey @plammel  corregido